mkdocs.yml

```
@@ -0,0 +1,74 @@
+site_name: Remote sensing deployable analysis tools (RS-DAT)
+repo_url: https://github.com/RS-DAT/RS-DAT
+repo_name: RS-DAT
+
+nav:
+  - What can RS-DAT do for you: index.md
+  - Deployable Analysis Tools:
+    - Jupyter Dask On SLURM: components/jupyter_dask_on_slurm.md
+    - Jupyter Dask On SRC: components/jupyter_dask_on_src.md
+    - Jupyter Dask Cloud to Cluster: components/jupyter_dask_cloud_to_cluster.md
+    - Jupyter Docker Spawner On SRC: components/jupyter_docker_spawner_on_src.md
+    - Jupyter Dask with Singularity: components/jupyterdask_with_singularity.md
+  - Examples: examples.md
+  - Demos:
+    - 2023-05-31-SummerSchool-HDCRS: demos/2023_05_31_SummerSchool_HDCRS.md
+    - 023-05-22-scalable-GIS: demos/023_05_22_scalable_GIS.md
+    - 2023-05-15-workshop-ITC: demos/2023_05_15_workshop_ITC.md
+    - 2023-03-10-demo-TU-Delft: demos/2023_03_10_demo_TU_Delft.md
+    - 2022-12-01-demo-remote-sensing-session: demos/2022_12_01_demo_remote_sensing_session.md
+  - How to contribute: how_to_contribute.md
+  - How to cite: how_to_cite.md
+  - License: what_license.md
```

The top bar seems quite busy - what do you think of simplifying some of the headers?

```suggestion
nav:
  - Introduction: index.md
  - Deploy:
    - Jupyter Dask On SLURM: components/jupyter_dask_on_slurm.md
    - Jupyter Dask On SRC: components/jupyter_dask_on_src.md
    - Jupyter Dask Cloud to Cluster: components/jupyter_dask_cloud_to_cluster.md
    - Jupyter Docker Spawner On SRC: components/jupyter_docker_spawner_on_src.md
    - Jupyter Dask with Singularity: components/jupyterdask_with_singularity.md
  - Examples: examples.md
  - Demos:
    - 2023-05-31-SummerSchool-HDCRS: demos/2023_05_31_SummerSchool_HDCRS.md
    - 023-05-22-scalable-GIS: demos/023_05_22_scalable_GIS.md
    - 2023-05-15-workshop-ITC: demos/2023_05_15_workshop_ITC.md
    - 2023-03-10-demo-TU-Delft: demos/2023_03_10_demo_TU_Delft.md
    - 2022-12-01-demo-remote-sensing-session: demos/2022_12_01_demo_remote_sensing_session.md
  - Contributing: how_to_contribute.md
  - Cite: how_to_cite.md
  - License: what_license.md
```